Kitodo/src/main/java/org/kitodo/production/model/bibliography/course/metadata/CountableMetadata.java

```diff
@@ -225,11 +251,18 @@ public String getValue(Pair<LocalDate, Issue> selectedIssue, MonthDay yearStart)
      * @return true, if the given metadataType and time point of creation match
      */
     public boolean matches(String metadataType, Pair<LocalDate, Issue> issue, Boolean create) {
-        return (metadataType == null || metadataType.equals(this.metadataType))
-                && (null == create && new IssueComparator(block).compare(this.create, issue) <= 0
+        if (metadataType == null || metadataType.equals(this.metadataType)) {
+            if (this.create.getRight()) {
+                return  Objects.nonNull(issue) && Objects.equals(this.create.getMiddle(), issue.getRight());
+            } else {
+                return (null == create
+                        && new IssueComparator(block).compare(Pair.of(this.create.getLeft(), this.create.getMiddle()), issue) <= 0
                         && (delete == null || new IssueComparator(block).compare(issue, delete) < 0)
                         || Boolean.TRUE.equals(create) && this.create.equals(issue) || Boolean.FALSE.equals(create)
-                                && (issue == null && this.delete == null || issue.equals(this.delete)));
+                        && (issue == null && this.delete == null || issue.equals(this.delete)));
```

Comment:
If it is possible replace null comparing with Objects.isNull() or Objects.nonNull() method calls.